WebContent/WEB-INF/classes/env.properties

```diff
@@ -74,6 +74,8 @@ abilit.USE_CACHE_DATA_SOURCES_POINTS_WHEN_THE_SYSTEM_IS_READY=true
 abilit.USE_ACL=false
 abilit.ACL_SERVER=http://localhost:8090
 
+#Events
+abilit.DONT_CREATE_EVETS_FOR_EMAIL_ERROR=false
```

Comment:
Is it possible to rename this flag , it means, by just put char 'N' :
DONT_CREATE_EVETS_FOR_EMAIL_ERROR  -> DONT_CREATE_EVENTS_FOR_EMAIL_ERROR

Reply:
Instead of adding char, "N" suggests DO_NOT_CREATE_EVETS_FOR_EMAIL_ERROR

Reply:
Ok